docs/src/SUMMARY.md

```diff
@@ -5,6 +5,7 @@
 # Tutorials
 
 - [Entry-level Model Checker Tutorial](./tutorials/entry-tutorial.md)
+- [Checking Pluscal specifications](./tutorials/pluscal-tutorial.md)
 - [Type Checker Tutorial](./tutorials/snowcat-tutorial.md)
 - [Symbolic Model Checking](./tutorials/symbmc.md)
```

Comment:
Adding this here makes it look like it's the second-most important tutorial, but it's rather technical in nature and many people won't care. Shall we move it somewhere else?

Reply:
I agree with this too. This is much more a technical workaround/appendix material imo.

Reply:
On one hand, yes. On the other hand, people are writing Pluscal specs.